node/cn/backend.go

```diff
@@ -215,19 +215,9 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	setEngineType(chainConfig)
 
 	// load governance state
+	chainConfig.SetDefaults()
+	// latest values will be applied to chainConfig after NewMixedEngine call
 	governance := governance.NewMixedEngine(chainConfig, chainDB)
-
-	// Set latest unitPrice/gasPrice
-	chainConfig.UnitPrice = governance.UnitPrice()
-	config.GasPrice = new(big.Int).SetUint64(chainConfig.UnitPrice)
```

Comment:
This line should not be removed. Only `chainConfig` is automatically up-to-date. `config` is not.